tests/acceptance/features/webUISharingExternal/federationSharing.feature

```diff
@@ -47,209 +47,78 @@ Feature: Federation Sharing - sharing with users on other cloud storages
     #    And folder "simple-folder (2)" should be listed on the webUI
     #    And folder "simple-empty-folder (2)" should be listed on the webUI
 
-  @skip @yetToImplement
   Scenario: declining a federation share on the webUI
-    Given user "user1" from server "REMOTE" has shared "/lorem.txt" with user "user1" from server "LOCAL"
+    Given user "user1" from remote server has shared "/lorem.txt" with user "user1" from local server
```

Comment:
```Given user "user1" from the remote server has shared "/lorem.txt" with user "user1" of the local server``` sounds better to me.

Reply:
This is a pre-existing step. So, good to be on a separate PR.